spec/thinking_sphinx/configuration/minimum_fields_spec.rb

```diff
@@ -6,7 +6,7 @@
   let(:indices) { [index_a, index_b] }
   let(:index_a) { double 'Index A', :model => model_a, :type => 'plain',
     :sources => [double(:fields => [field_a1, field_a2])] }
-  let(:index_b) { double 'Index B', :model => model_a, :type => 'rt',
+  let(:index_b) { double 'Index B', :model => model_b, :type => 'rt',
```

Comment:
This got me in trouble originally. `index_a` should use `model_a` and `index_b` should use `model_b` as far as I understand the intention of the test case.

Reply:
As much as I can tell, yes, I think you're right! Thanks for this PR :)